scipy/optimize/_milp.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 from scipy.sparse import csc_array, vstack, issparse
 from scipy._lib._util import VisibleDeprecationWarning
-from ._highs._highs_wrapper import _highs_wrapper  # type: ignore[import-not-found]
+from ._highs._highs_wrapper import _highs_wrapper  # type: ignore[import-untyped]
```

Comment:
CI failure is:

```
scipy/optimize/_milp.py:5: error: Cannot find implementation or library stub for module named "scipy.optimize._highs._highs_wrapper"  [import-not-found]
scipy/optimize/_milp.py:5: note: Error code "import-not-found" not covered by "type: ignore" comment
scipy/optimize/_milp.py:5: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 888 source files)
```

Reply:
I'm seeing `import-untyped` with the same Mypy version - and that's clearly more correct since `_highs_wrapper` does exist. Probably yet another Mypy bug that I don't want to know about, so I silenced both error codes now.